features/data_converter/empty/feature.go

```diff
@@ -0,0 +1,55 @@
+package empty
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	historyProto "go.temporal.io/api/history/v1"
+	"go.temporal.io/sdk-features/harness/go/harness"
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/workflow"
+)
+
+var Feature = harness.Feature{
+	Workflows:   Workflow,
+	CheckResult: CheckResult,
+}
+
+// run a workflow that returns nothing
+func Workflow(ctx workflow.Context) error {
+	return nil
+}
+
+func CheckResult(ctx context.Context, runner *harness.Runner, run client.WorkflowRun) error {
+	// verify client result is null
+	var result interface{}
+	if err := run.Get(ctx, &result); err != nil {
+		return err
+	}
+	if result != nil {
+		return fmt.Errorf("invalid result: %v", result)
+	}
+	history := runner.Client.GetWorkflowHistory(ctx, run.GetID(), "", false, 0)
+
+	var attrs *historyProto.WorkflowExecutionCompletedEventAttributes
+
+	for history.HasNext() {
+		ev, err := history.Next()
+		if err != nil {
+			return err
+		}
+		// get result payload of WorkflowExecutionCompleted event from workflow history
+		attrs = ev.GetWorkflowExecutionCompletedEventAttributes()
+		if attrs != nil {
+			break
+		}
+	}
+	if attrs == nil {
+		return errors.New("could not locate WorkflowExecutionCompleted event")
+	}
+
+	// verify the workflow execution completed event has no payload
+	runner.Require.Empty(attrs.GetResult().GetPayloads())
```

Comment:
If you use `runner.Require.Empty` here, you might as well use `runner.Require.NotNil` for your result check above.